RevenueCatUI/CustomerCenter/ViewModels/ManageSubscriptionsViewModel.swift

```diff
@@ -27,21 +27,33 @@ import SwiftUI
 class ManageSubscriptionsViewModel: ObservableObject {
 
     let screen: CustomerCenterConfigData.Screen
-    let paths: [CustomerCenterConfigData.HelpPath]
+
+    var relevantPathsForPurchase: [CustomerCenterConfigData.HelpPath] {
+        if purchaseInformation?.explanation == .lifetime {
```

Comment:
I think this is a good proxy, but not bulletproof if we add another explanation that is also used for lifetime purchases. Maybe we can add another property to `purchaseInformation` that represents for real if it's a lifetime product